### ISSUE TITLE
[GUI] Payment codes

### ISSUE THREAD

**Issue by pythcoiner** [wizardsardine/liana#1043]
We should accept payments (`bitcoin:<address>?amount=<amount>`) codes in adress input

**Comment by darosior** [comment]
You mean URIs?

**Comment by darosior** [comment]
I think it would be confusing that you can paste the amount inside the address text input..

**Comment by pythcoiner** [comment]
yes i mean URI, what i think we should do in this case is to extract adress and amount and fill each TextInput w/ the relevant data.
It can be usefull as some payment processor give you this URI as default when you click their `copy` address button (BTCPay IIRC), it also avoid footguns(wrong amount input by user) as some services will detect a received payment only if the exact amount is received.
